scripts/start_daemon.py

```
@@ -15,10 +15,18 @@
-sys.path.insert(0, str(Path(__file__).parent.parent))
+_AITEXT_ROOT = Path(__file__).resolve().parent.parent
+if str(_AITEXT_ROOT) not in sys.path:
+    sys.path.insert(0, str(_AITEXT_ROOT))
+
+try:
+    from load_env import load_env
+
+    load_env()
+except Exception:
+    # 非标准启动方式或缺少 .env 时忽略
+    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)